### ISSUE TITLE
Erratic readings from hx711

### ISSUE THREAD

**Issue by JoeToeniskoetter** [dangrie158/opencushion#8]
I am using the proved example and I am getting readings on the screen however, my readings are very erratic. 
Here is what I get in the console when running the example code:
0.0575999990105629
0.003599999938160181
-2.8401999473571777
0.00800000037997961
0.04340000078082085
0.05620000138878822
-1.393399953842163
0.016599999740719795
2.8845999240875244
2.8296000957489014
-1.4026000499725342

Any help would be appreciated. 
Thanks!

**Comment by alina** [comment]
Having the same issue:

16.774999618530273
-8.684399604797363
-5.259200096130371
16.5802001953125
-8.912799835205078
-7.584199905395508
-2.6602001190185547
-15.50100040435791
-61.790199279785156
1.4910000562667847

I verified that it's not a sensor problem with a python hx711 library which worked fine. Any suggestions would be much appreciated. Thank you!

**Comment by SimonHLMC** [comment]
Did you solve the Problem? I have got the same.

**Comment by JoeToeniskoetter** [comment]
> Did you solve the Problem? I have got the same.

@Swompet 

Yes I did end up fixing my issues Which were caused by incorrect wiring. Double and triple check your wiring. It can be very easy to mess up and connect the wires incorrectly.

**Comment by JoeToeniskoetter** [comment]
> Did you solve the Problem? I have got the same.

@Swompet 

Yes I did end up fixing my issues Which were caused by incorrect wiring. Double and triple check your wiring. It can be very easy to mess up and connect the wires incorrectly.

**Comment by SimonHLMC** [comment]
@JoeToeniskoetter 
The thing is this works fine with a python script. What wiring did you mess up?

**Comment by JoeToeniskoetter** [comment]
@Swompet 
The wires connecting my 4 load cells into a Wheatstone bridge were wrong. So when using your python script you are getting correct readings from the hx711?

**Comment by SimonHLMC** [comment]
@JoeToeniskoetter Yes in python it works fine. did you do anything with the bit order?

**Comment by SimonHLMC** [comment]
I was able to fix this. I was a pin issue.
But now I am not able to calibrate the cell or tare it.
this is my output: 
at some point i pressed tare.
Units  : 28202.230469 | Offset: 0
Units  : 31721.539062 | Offset: 0
Units  : 28193.691406 | Offset: 0
Units  : 35236.230469 | Offset: 0
Units  : 35232.460938 | Offset: 0
Units  : 35231.308594 | Offset: 0
Units  : 35230.769531 | Offset: 0
Units  : 35228.691406 | Offset: 0
Units  : 35228.847656 | Offset: 0
Units  : 35230.769531 | Offset: 0 //////////at this point i pressed the tare button
Units  : 1.615385 | Offset: -458029
Units  : 4.153846 | Offset: -458029
Units  : 6.461538 | Offset: -458029
Units  : 10.769231 | Offset: -458029
Units  : -3507.538574 | Offset: -458029
Units  : 28.384615 | Offset: -458029
Units  : 26.769230 | Offset: -458029
Units  : 30.769230 | Offset: -458029
Units  : 37.769230 | Offset: -458029
Units  : 41.384617 | Offset: -458029
Units  : 46.538460 | Offset: -458029
Units  : 49.076923 | Offset: -458029
Units  : 54.923077 | Offset: -458029
Units  : 56.846153 | Offset: -458029
Units  : 64.230766 | Offset: -458029
Units  : -3461.846191 | Offset: -458029
Units  : 70.769234 | Offset: -458029
Units  : 70.076920 | Offset: -458029
Units  : 76.769234 | Offset: -458029
Units  : 78.230766 | Offset: -458029
Units  : 82.384613 | Offset: -458029
Units  : 82.153847 | Offset: -458029
Units  : 84.384613 | Offset: -458029
Units  : 82.692307 | Offset: -458029
Units  : 81.000000 | Offset: -458029
Units  : 82.076920 | Offset: -458029
Units  : -3445.923096 | Offset: -458029
Units  : 85.923080 | Offset: -458029
Units  : 87.307693 | Offset: -458029
Units  : -3444.692383 | Offset: -458029
Units  : -3440.769287 | Offset: -458029
Units  : 90.307693 | Offset: -458029
Units  : 17745.691406 | Offset: -367384
Units  : 7061.538574 | Offset: -367384
Units  : 7034.077148 | Offset: -367384
Units  : 23.769230 | Offset: -366992
